revup/github_utils.py

```diff
@@ -298,8 +302,11 @@ async def query_everything(
                 pr_labels.add(label["name"])
                 pr_label_ids.add(label["id"])
             for revs in this_node["reviewRequests"]["nodes"]:
-                reviewers.add(revs["requestedReviewer"]["login"])
-                reviewer_ids.add(revs["requestedReviewer"]["id"])
+                if not revs["requestedReviewer"]:
+                    continue
+                elif "login" in revs["requestedReviewer"]:
+                    reviewers.add(revs["requestedReviewer"]["login"])
+                    reviewer_ids.add(revs["requestedReviewer"]["id"])
```

Comment:
A little confused by this. Don't you still need to add the team as a reviewer if there's no `login`? (my understanding is that the team would have a `name` and `id` but no `login`)